### PR TITLE
Feature/nuke upstream 20jan

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
Relates to #101 
- Nuke overrides from core

### CODE REVIEW

File: avalon/tools/loader/widgets.py [hunk 1]
```diff
@@ -522,7 +522,7 @@ def sorter(value):
                 error_info.append((
                     "Incompatible Loader",
                     None,
-                    representation["name"],
+                    representation.get("name"),
```
Comment (iLLiCiTiT) [inline avalon/tools/loader/widgets.py:525]: This is already fixed in `2.x/develop` (and not like this).

File: avalon/nuke/workio.py [hunk 2]
```diff
@@ -1,11 +1,10 @@
 """Host API required Work Files tool"""
 import os
 import nuke
-from avalon import api
 
 
 def file_extensions():
-    return api.HOST_WORKFILE_EXTENSIONS["nuke"]
+    return [".nk"]
```
Comment (iLLiCiTiT) [inline avalon/nuke/workio.py:7]: should use `return api.HOST_WORKFILE_EXTENSIONS["nuke"]`

File: avalon/nuke/pipeline.py [hunk 3]
```diff
@@ -279,18 +266,9 @@ def _install_menu():
         publish,
         workfiles,
         loader,
-        sceneinventory,
-        libraryloader
```
Comment (iLLiCiTiT) [inline avalon/nuke/pipeline.py:283]: Maybe we should keep `libraryloader` here (if is not overriden with ours menu in pype).

File: avalon/nuke/pipeline.py [hunk 4]
```diff
@@ -165,54 +154,37 @@ class Creator(api.Creator):
     """
     node_color = "0xdfea5dff"
 
-    def __init__(self, *args, **kwargs):
```
Comment (iLLiCiTiT) [inline avalon/nuke/pipeline.py:168]: Not sure if this was important?

Comment (mkolar) [reply avalon/nuke/pipeline.py:168]: I wasn't able to make it work with this on. We have further changes to this in our code and it was quite a stopper.

File: avalon/nuke/pipeline.py (hunk 3 above)
Comment (mkolar) [reply avalon/nuke/pipeline.py:283]: we override menu completery

File: avalon/nuke/workio.py (hunk 2 above)
Comment (mkolar) [reply avalon/nuke/workio.py]: agreed

File: avalon/tools/loader/widgets.py (hunk 1 above)
Comment (mkolar) [reply avalon/tools/loader/widgets.py]: thanks. fixed

**Review by mkolar — APPROVED** [review]
works well now

**Pull request merged**